app/presenters/project_dashboard_presenter.rb

```diff
@@ -46,11 +46,21 @@ def role(user)
   end
 
   def quota_usage(session_id:)
-    quota_usage = "#{project.storage_usage(session_id:)} out of #{project.storage_capacity(session_id:)} used"
+    if project.pending?
+      quota_usage = "0 KB out of 0 GB used"
+    else
+      quota_usage = "#{project.storage_usage(session_id:)} out of #{project.storage_capacity(session_id:)} used"
+    end
     quota_usage
   end
 
   def quota_percentage(session_id:)
-    # figure out how to calculate percentage of storage used with different and unpredictable units
+    q_percent = project.storage_capacity_raw(session_id:).zero? ? 0 : (project.storage_usage_raw(session_id:).to_f / project.storage_capacity_raw(session_id:).to_f) * 100
+
+    if project.pending?
+      0
+    else
+      q_percent
+    end
```

Comment:
It took me a bit to parse the code.  Maybe something like below?

```suggestion
    return 0 if project.pending? || project.storage_capacity_raw(session_id:).zero?
    
   (project.storage_usage_raw(session_id:).to_f / project.storage_capacity_raw(session_id:).to_f) * 100
```